src/components/Calculator.js

```diff
@@ -1,64 +1,58 @@
 /* eslint-disable */
```

Comment:
- [ ] It is a good practice to fix the linter errors manually instead of disabling them. Please remove this line and fix the linter errors manually.